Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/Microsoft365DefenderEventCollector/Microsoft365DefenderEventCollector.py

```diff
@@ -328,30 +330,35 @@ def test_module(get_events: DefenderGetEvents) -> str:
     :return: 'ok' if test passed, anything else will fail the test.
     :rtype: ``str``
     """
-
-    message: str = ''
     try:
         get_events.client.request.params = {'limit': 1}
         get_events.run()
-        message = 'ok'
+        return 'ok'
     except DemistoException as e:
         if 'Forbidden' in str(e) or 'authenticate' in str(e):
-            message = AUTH_ERROR_MSG
-        else:
-            raise
-    return message
+            return AUTH_ERROR_MSG
+        raise
 
 
-def main(command: str, demisto_params: dict):
+def main(command: str, params: dict):
     demisto.debug(f'Command being called is {command}')
     try:
+        params_endpoint_type = params.get('endpoint_type')
+        params_url = params.get('url')
+        is_gcc = params.get('is_gcc')
+        endpoint_type, params_url = microsoft_defender_for_endpoint_get_base_url(is_gcc, params_endpoint_type, params_url)
```

Comment:
```suggestion
        endpoint_type, params_url = microsoft_defender_for_endpoint_get_base_url(None, params_endpoint_type, params_url)
```

** OR - to change the function declaration:
def microsoft_defender_for_endpoint_get_base_url(params_endpoint_type, params_url, is_gcc=None):

Reply:
fixed